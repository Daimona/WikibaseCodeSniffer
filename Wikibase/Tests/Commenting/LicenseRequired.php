<?php

class ClassWithNoLicense {
}

/**
 * Very important class-level comment
 */
class ClassWithCommentButNoLicense {
}

/**
 * @license TESTTESTTEST
 */
class ClassWithoutRequiredLicense {
}