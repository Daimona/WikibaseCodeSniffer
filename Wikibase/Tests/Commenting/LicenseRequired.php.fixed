<?php

/**
 * @license GPL-2.0-or-later
 */
class ClassWithNoLicense {
}

/**
 * Very important class-level comment
 *
 * @license GPL-2.0-or-later
 */
class ClassWithCommentButNoLicense {
}

/**
 * @license TESTTESTTEST
 * @license GPL-2.0-or-later
 */
class ClassWithoutRequiredLicense {
}