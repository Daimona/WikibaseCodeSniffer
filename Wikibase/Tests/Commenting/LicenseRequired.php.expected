  3 | ERROR | Class level documentation missing
  9 | ERROR | No @license tag: GPL-2.0-or-later found
 15 | ERROR | No @license tag: GPL-2.0-or-later found
